src/components/icons/algolia-logo/index.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function AlgoliaLogo(props) {
+    const { className, style } = props;
+    return (
+        <div className={className} style={style}>
+            <span>Search by</span>{' '}
+            <a
+                href="https://www.algolia.com/?utm_source=react-instantsearch&amp;utm_medium=website&amp;utm_content=react-instantsearch.netlify.com&amp;utm_campaign=poweredby"
+                target="_blank"
+                className="ais-PoweredBy-link"
+                aria-label="Algolia"
+                rel="noopener noreferrer"
+            >
+                <svg
```

Comment:
under icons, we should only store the icon and then all other parts are a component under `src/components`